aspnetcore/security/authentication/other-logins.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Short survey of other login providers.
+author: rick-anderson
+ms.author: riande
+manager: wpickett
+ms.date: 11/3/2016
+ms.topic: article
+ms.assetid: BC36CA84-3DE8-496E-9AA2-2F1B74AE8309
+ms.prod: aspnet-core
+﻿uid: security/authentication/sociallogins
+---
+# Short survey of other login providers
+
+<a name=security-authentication-other-logins></a>
```

Comment:
If we keep this we should have one one sample using just the OAuth base class.
https://github.com/aspnet/Security/blob/dev/samples/SocialSample/Startup.cs#L201-L271